iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template

```diff
@@ -742,6 +742,11 @@ buffered_arrays_memory_proportion=0.6
 # Datatype: double
 reject_proportion=0.8
 
+# Ratio of max memory size for the WAL buffer queue, 0.1 by default
+# effectiveMode: restart
```

Comment:
Ratio to what (whose 10%)? Should be more precise.

Reply:
Let me fix it.